canonical-web-and-design/canonical.com
# Add functionality to /partners/become-a-partner

## Done
- Add functionality and copy update for `/partners/become-a-partner` page
- Add solid background to `careers/start` page
- Remove lorem ipsum job description on all `careers` pages


## QA
- Run the website locally using `./run`
- Go to [/partners/become-a-partner](http://0.0.0.0:8002/partners/become-a-partner), compare to [copy doc](https://docs.google.com/document/d/1Ihh9s3i72saCYaheXRVRn1Meg5lYN5U1gFbdvLMdWcc/edit#) and check form functionality. Soon I will add a TY page
- Go to [/careers/start](http://0.0.0.0:8002/careers/start) and make sure the background is changed and the page looks nice


## Issue
Fixes https://github.com/canonical-web-and-design/web-squad/issues/1842
